src/driver.rs

```diff
@@ -292,3 +269,36 @@ pub fn main() {
         }
     }))
 }
+
+#[must_use]
+fn help_message() -> &'static str {
+    color_print::cstr!(
+"Checks a package to catch common mistakes and improve your Rust code.
+
+<green,bold>Usage</>:
+    <cyan,bold>cargo clippy</> <cyan>[OPTIONS] [--] [<<ARGS>>...]</>
```

Comment:
```suggestion
    <cyan,bold>clippy-driver</> <cyan>[OPTIONS] [--] [<<ARGS>>...]</>
```

This whole message seems to be copy pasted from the `cargo clippy` command. I don't think they are the same and support the same flags. Does `clippy-driver` really support `--fix` for example? :thinking:  Also, I don't think we accept anything (special) after `--` in the clippy driver.